cms/templates/howitworks.html

```diff
@@ -14,8 +16,8 @@
 <div class="wrapper-content-header wrapper">
   <section class="content content-header">
     <header>
-      <h1><span class="wrapper-text-welcome">${_("Welcome to {studio_name}").format(
-        studio_name=u'</span><span class="logo">{studio_name}</span>'.format(
+      <h1><span class="wrapper-text-welcome">${Text(_("Welcome to {studio_name}")).format(
+        studio_name=HTML(u'</span><span class="logo">{studio_name}</span>').format(
```

Comment:
@cptvitamin Thanks for looking at this.

You can load the page by signing out of the studio. And visit home url ( https://studio.edx.org/ )